slug_test.go

```diff
@@ -58,25 +57,12 @@ func TestPack(t *testing.T) {
 			}
 			symFound = true
 		}
-
-		if hdr.Name == "dangling_symlink" {
```

Comment:
I'm still trying to parse the tests, but is this case still handled by the new ones?

Reply:
ah never mind, I see the `nonexistent` test cases now